cmd/buildkitd/config/config.go

```diff
@@ -27,6 +27,11 @@ type Config struct {
 	DNS *DNSConfig `toml:"dns"`
 
 	History *HistoryConfig `toml:"history"`
+
+	// LogFormat is the format of the logs. It can be "json" or "text".
+	Log struct {
```

Comment:
Please avoid nested struct and create a dedicated type `LogConfig` like `HistoryConfig`.

Reply:
Done.
Out of curiosity what is the rational for avoiding nested struct?
I saw the `Workers` nested struct and thought to do the same because `Log` would be a simple one.